streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -17,17 +17,20 @@
 
 import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.streams.processor.StateStoreSupplier;
+import org.apache.kafka.streams.state.KeyValueStore;
+import org.apache.kafka.streams.state.WindowStore;
 
 /**
  * {@link KGroupedStream} is an abstraction of a <i>grouped record stream</i> of key-value pairs
  * usually grouped on a different key than the original stream key
- *
+ * <p>
```

Comment:
probably no. I think thats because of auto indent (in IDE) that I am using after each change.